### PR TITLE
Pass prerelease option to nightly builds

### PR DESCRIPTION
**Pull request by r0qs**
Updates the CI job to use `emscripten_build` script with `prerelease` parameter introduced by https://github.com/ethereum/solidity/pull/13581.

Depends on https://github.com/ethereum/solidity/pull/13581.

### CODE REVIEW

**Comment by cameel** [comment]
@r0qs Do you need permissions access to solc-bin? We have same problem as last time, i.e. the PR checks won't pass because this is an external PR.

I actually haven't considered that this will affect all contributions. In that case we might want to fix that after all. See https://github.com/ethereum/solidity/issues/13614.

For now I'll just force-merge this one (once the dependency in the main repo is merged).

**Review by cameel — COMMENTED** [review]
Looks good but not approving yet since I have not yet reviewed the dependency in the main repo.

**Comment by r0qs** [comment]
> @r0qs Do you need permissions access to solc-bin? We have same problem as last time, i.e. the PR checks won't pass because this is an external PR.

No, I have permission. Sorry, it was my mistake, I set two git remote trackers in my fork and pushed the branch to the wrong one, haha
I just realized that I did the same for the other PR in the solidity repo. Should I recreate the PR?

> I actually haven't considered that this will affect all contributions. In that case we might want to fix that after all. See [ethereum/solidity#13614](https://github.com/ethereum/solidity/issues/13614).

Ah, indeed.

**Comment by cameel** [comment]
> Should I recreate the PR?

Like I said, I can just force merge it so no need.

**Comment by cameel** [comment]
@r0qs This needs to be updated to match https://github.com/ethereum/solidity/pull/13581 (and probably also rebased).

**Review by cameel — APPROVED** [review]

**Pull request merged**
